### PR TITLE
[SW-1509] Rename  colsampleBytree to  colSampleByTree on H2OXGBoost in Scala & Python

### PR DESCRIPTION
**Pull request by jakubhava**

### CODE REVIEW

File: py/ai/h2o/sparkling/ml/params/H2OXGBoostParams.py [hunk 1]
```diff
@@ -311,8 +312,12 @@ def getColSampleByLevel(self):
     def getColSampleRatePerTree(self):
         return self.getOrDefault(self.colSampleRatePerTree)
 
-    def getColsampleBytree(self):
-        return self.getOrDefault(self.colsampleBytree)
+    def getColsampleByTree(self):
```
Comment (mn-mikke) [inline py/ai/h2o/sparkling/ml/params/H2OXGBoostParams.py]: Shouldn't it be `getColSampleByTree`?

File: py/ai/h2o/sparkling/ml/params/H2OXGBoostParams.py [hunk 2]
```diff
@@ -311,8 +312,12 @@ def getColSampleByLevel(self):
...
+        return self.getOrDefault(self.colSampleByTree)
+
+    def getColsampleByTree(self):
```
Comment (mn-mikke) [inline py/ai/h2o/sparkling/ml/params/H2OXGBoostParams.py]: Shouldn't it be `getColsampleBytree`?

File: py/ai/h2o/sparkling/ml/params/H2OXGBoostParams.py [hunk 3]
```diff
@@ -434,8 +439,12 @@ def setColSampleByLevel(self, value):
     def setColSampleRatePerTree(self, value):
         return self._set(colSampleRatePerTree=value)
 
+    def setColSampleByTree(self, value):
+        return self._set(colSampleBytree=value)
```
Comment (mn-mikke) [inline py/ai/h2o/sparkling/ml/params/H2OXGBoostParams.py]: `self._set(colSampleByTree=value)`?

File: ml/src/main/scala/ai/h2o/sparkling/ml/algos/H2OXGBoost.scala [hunk 4]
```diff
@@ -168,7 +168,10 @@ trait H2OXGBoostParams extends H2OAlgoSupervisedParams[XGBoostParameters] {
 
   def getColSampleRatePerTree(): Double = $(colSampleRatePerTree)
 
-  def getColsampleBytree(): Double = $(colsampleBytree)
+  def getColSampleByTree(): Double = $(colSampleByTree)
+
+  @DeprecatedMethod("getColSampleBytree")
```
Comment (mn-mikke) [inline ml/src/main/scala/ai/h2o/sparkling/ml/algos/H2OXGBoost.scala]: `"getColSampleByTree"`?

**Comment by jakubhava** [comment]
Done, thanks for the review

**Pull request merged**